src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -2083,17 +2087,20 @@ void ApplicationManagerImpl::OnServiceStartedCallback(
   SDL_LOG_DEBUG("ServiceType = " << type << ". Session = " << std::hex
                                  << session_key);
   std::vector<std::string> empty;
+  std::string reason;
```

Comment:
@ychernysheva const

Reply:
@AKalinich-Luxoft This reason cannot be const, because it may be re-assigned here to valid NACK reason (related to disallowed HMI type)